hugo/content/courses/javascript/interview-graphs.md

```diff
@@ -120,13 +109,10 @@ function dfs(start, visited = new Set()) {
             console.log(`DFS found Bangkok`)
             return;
         }
-        
```

Comment:
removing this is a stretch

Reply:
removing this space ? what do you mean by "stretch" ?